Commands/gemini3.py

```
@@ -148,20 +116,31 @@ def reply_with_grounded_gemini(self, message):
-    result = generate(prompt, grounding_text=body_content)
-    prefix = "🔎 Grounded: " if body_content.strip() else "Not Grounded: "
+    is_grounded = bool(duck_urls)
+
+    result = gemini_generate({
+        "prompt": prompt,
+        "grounded": is_grounded,
+        "grounding_text": grounding_text
+    }, model)
 
     if "Error" in result[0]:
         self.send_privmsg(channel, f"Failed to generate a response. Please try again later.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)